vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -286,7 +282,6 @@ def save_custom_attributes_inventory(parent, hashes)
   end
 
   def save_filesystems_inventory(parent, hashes)
-    return if hashes.nil?
```

Comment:
I see this change but don't see a corresponding change in the model (which I expected)...is that correct?